<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ususarios</class>
 <widget class="QTabWidget" name="Ususarios">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuarios</string>
  </property>
  <widget class="QWidget" name="General">
   <attribute name="title">
    <string>General</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0">
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEditCp"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Correo</string>
        </property>
        <property name="buddy">
         <cstring>lineEditMail</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditUsusario"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>C.P.</string>
        </property>
        <property name="buddy">
         <cstring>lineEditCp</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Cla&amp;ve</string>
        </property>
        <property name="buddy">
         <cstring>lineEditClave</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&amp;Usuario</string>
        </property>
        <property name="buddy">
         <cstring>lineEditUsusario</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&amp;Nombre</string>
        </property>
        <property name="buddy">
         <cstring>lineEditNombre</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>NIF</string>
        </property>
        <property name="buddy">
         <cstring>lineEditNif</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Telefono</string>
        </property>
        <property name="buddy">
         <cstring>lineEditTlfn</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditTlfn">
        <property name="inputMask">
         <string>999-99-99-99</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="lineEditClave">
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNombre"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Per&amp;misos</string>
        </property>
        <property name="buddy">
         <cstring>lineEditPermisos</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditProvincia"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Provincia</string>
        </property>
        <property name="buddy">
         <cstring>lineEditProvincia</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLineEdit" name="lineEditNif">
        <property name="inputMask">
         <string>99.999.999-&gt;A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&amp;Localidad</string>
        </property>
        <property name="buddy">
         <cstring>lineEditLocalidad</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Direccion</string>
        </property>
        <property name="buddy">
         <cstring>lineEditDireccion</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditDireccion"/>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditMail"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Apellidos</string>
        </property>
        <property name="buddy">
         <cstring>lineEditApellido</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="6">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEditLocalidad"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Notas</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLineEdit" name="lineEditArchivoFoto">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="pushButtonCargarImagen">
        <property name="text">
         <string>Cargar Imagen</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="pushButtonBorrarImagen">
        <property name="text">
         <string>Borrar Imagen</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditPermisos"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEditApellido"/>
      </item>
      <item row="0" column="4" rowspan="7" colspan="2">
       <widget class="QLabel" name="labelFoto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string>Sin foto</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignJustify|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Codigo</string>
        </property>
        <property name="buddy">
         <cstring>lineEditCod</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditCod">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelNombre">
        <property name="font">
         <font>
          <family>Bitstream Charter</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Nombre de usuario</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonNuevo">
        <property name="text">
         <string>Nuevo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonModificar">
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonBorrar">
        <property name="text">
         <string>Borrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRefrescar">
        <property name="text">
         <string>Refrescar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAnterior">
        <property name="text">
         <string>Anterior</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSiguiente">
        <property name="text">
         <string>Siguiente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCerrar">
        <property name="text">
         <string>Cerrar</string>
        </property>
        <property name="shortcut">
         <string>Esc</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Tab 2</string>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditCod</tabstop>
  <tabstop>lineEditNombre</tabstop>
  <tabstop>lineEditApellido</tabstop>
  <tabstop>lineEditDireccion</tabstop>
  <tabstop>lineEditLocalidad</tabstop>
  <tabstop>lineEditProvincia</tabstop>
  <tabstop>lineEditCp</tabstop>
  <tabstop>lineEditTlfn</tabstop>
  <tabstop>lineEditNif</tabstop>
  <tabstop>lineEditMail</tabstop>
  <tabstop>lineEditUsusario</tabstop>
  <tabstop>lineEditClave</tabstop>
  <tabstop>pushButtonCargarImagen</tabstop>
  <tabstop>pushButtonBorrarImagen</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>lineEditArchivoFoto</tabstop>
  <tabstop>pushButtonNuevo</tabstop>
  <tabstop>pushButtonModificar</tabstop>
  <tabstop>pushButtonBorrar</tabstop>
  <tabstop>pushButtonRefrescar</tabstop>
  <tabstop>pushButtonAnterior</tabstop>
  <tabstop>pushButtonSiguiente</tabstop>
  <tabstop>pushButtonCerrar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCerrar</sender>
   <signal>clicked()</signal>
   <receiver>Ususarios</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1036</x>
     <y>755</y>
    </hint>
    <hint type="destinationlabel">
     <x>1098</x>
     <y>742</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
